x-pack/libbeat/autodiscover/providers/aws/elb/_meta/fields.yml

```diff
@@ -0,0 +1,84 @@
+- key: elb_listener
```

Comment:
Should wecall it just `elb`?

Reply:
An ELB is the load balancer, but each port it exposes is an ELB Listener. 

This provider actually emits ELB Listeners, because presumable you want to monitor each port, not each ELB. Orienting it around the listeners means you get all the right metadata like port etc.